bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/GarageDoorData.java

```diff
@@ -158,31 +142,80 @@ public String getDeviceName() {
 		return this.DeviceName;
 	}
 
-	public int getStatus() {
+	public GarageDoorStatus getStatus() {
 		return this.Status;
 	}
 
-	public boolean IsDoorClosed() {
-		if (this.Status == 2) {
-			return true;
+	public enum GarageDoorStatus {
```

Comment:
Not sure how much you have used Java enums, if you have not then I'll I can say is they are a little syntactically odd compared to ones in C or other languages, but they work well for modeling this kind of stuff.